### PR TITLE
Include Credit Key icon

### PR DESCRIPTION
**Pull request by ryanas**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by larouxn** [comment]
👋 @ryanas, looks like the SVG's title needs to be updated. Looks like it will also need to be optmized with svgo ([step 5 in here](https://github.com/activemerchant/payment_icons/blob/17bc4291a913d3bc178b78c25bbafff0b8733401/CONTRIBUTING.md)). The [template](https://github.com/activemerchant/payment_icons/blob/17bc4291a913d3bc178b78c25bbafff0b8733401/CONTRIBUTING.md#template) may also prove helpful.

```ruby
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:78]:
{:message=>"The 'creditkey' SVG file does not have the appropriate <title> value"}.
Expected: "Credit Key"
  Actual: "{Credit Key}"
```

If you'd like to run theses tests locally you can run `bundle exec rails test`.

**Comment by larouxn** [comment]
👋 CI still says title is off btw

```ruby
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:81]:
{:message=>"The 'creditkey' SVG file does not have the appropriate 'id' value on the <title> tag"}.
Expected: "pi-creditkey"
  Actual: "pi-Credit-Key"
```

**Review by larouxn — CHANGES_REQUESTED** [review]
👋 Looks like the `title` tag is incorrect.

```ruby
# Running:
.F
/home/runner/work/payment_icons/payment_icons/vendor/bundle/ruby/2.5.0/gems/railties-5.0.7.2/lib/rails/test_unit/reporter.rb:70:in `method': undefined method `test_Every_payment_SVG_meets_accessibility_requirements' for class `Minitest::Result' (NameError)
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:81]:
{:message=>"The 'creditkey' SVG file does not have the appropriate 'id' value on the <title> tag"}.
Expected: "pi-creditkey"
  Actual: "pi-creditKey"
```

**Comment by cjdeg** [comment]
Hi. Any update on the changes requested? Thanks!

**Comment by larouxn** [comment]
:wave: There are some failing tests. Can we either update or remove the fill `id`? ([CI logs](https://github.com/activemerchant/payment_icons/runs/7429435920?check_suite_focus=true))

```ruby
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:89]:
	from /home/runner/work/payment_icons/payment_icons/vendor/bundle/ruby/2.5.0/gems/minitest-5.15.0/lib/minitest.rb:874:in `block in record'
{:message=>"The 'Fill-1' ID should be pi-creditkey-Fill-1 (missing 'pi-' prefix)"}.
Expected /pi-(.*)/ to match "Fill-1".
```

**Comment by cjdeg** [comment]
@larouxn  is there any way to review the recent change to this PR? I’m running out of time to switch merchants over to this new app. Thanks

**Comment by cjdeg** [comment]
> 👋 There are some failing tests. Can we either update or remove the fill `id`? ([CI logs](https://github.com/activemerchant/payment_icons/runs/7429435920?check_suite_focus=true))
> 
> ```ruby
> Failure:
> PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:89]:
> 	from /home/runner/work/payment_icons/payment_icons/vendor/bundle/ruby/2.5.0/gems/minitest-5.15.0/lib/minitest.rb:874:in `block in record'
> {:message=>"The 'Fill-1' ID should be pi-creditkey-Fill-1 (missing 'pi-' prefix)"}.
> Expected /pi-(.*)/ to match "Fill-1".
> ```

Hi Nicholas. We made the requested change. Could someone possibly review this PR? Thanks!

**Comment by cjdeg** [comment]
@larouxn Hi. Any update on this change? Thanks!

**Review by larouxn — APPROVED** [review]
Looks good to me 🎩 
Screen Shot 2022-08-09 at 12 26 13

File: app/assets/images/payment_icons/creditkey.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-creditkey"><title id="pi-creditkey">Credit Key</title><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M19.3 11.7v-4h4c0 2.2-1.9 4-4 4zm0 4.4v-4c2.2 0 4 1.7 4 4h-4zm-.6-4.4h-3.8c0-2.2 1.7-4 3.8-4v4zm0 4.4c-2.2 0-4-1.7-4-4h4v4zM25 6.8l-6-3.6-6 3.6c-.9.5-1.4 1.4-1.4 2.4v5.5c0 1 .5 1.9 1.4 2.4l6 3.6 6-3.6c.9-.5 1.4-1.4 1.4-2.4V9.3c0-1.1-.5-1.9-1.4-2.5z" fill="#3d9ce5"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/creditkey.svg:1]: There's a duplicated border so the border color looks darker. We will need to remove it.

[video demonstration](https://screenshot.click/09-02-bzud6-obtta.mp4)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-creditkey"><title id="pi-creditkey">Credit Key</title><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M19.3 11.7v-4h4c0 2.2-1.9 4-4 4zm0 4.4v-4c2.2 0 4 1.7 4 4h-4zm-.6-4.4h-3.8c0-2.2 1.7-4 3.8-4v4zm0 4.4c-2.2 0-4-1.7-4-4h4v4zM25 6.8l-6-3.6-6 3.6c-.9.5-1.4 1.4-1.4 2.4v5.5c0 1 .5 1.9 1.4 2.4l6 3.6 6-3.6c.9-.5 1.4-1.4 1.4-2.4V9.3c0-1.1-.5-1.9-1.4-2.5z" fill="#3d9ce5"/></svg>
```

**Review by tauthomas01 — CHANGES_REQUESTED** [review]

**Review by tauthomas01 — APPROVED** [review]

**Comment by cjdeg** [comment]
@larouxn @tauthomas01, when can we expect this PR to be merged? Can we expect the payment method icon for our app to automatically switch to using the Credit Key icon?

**Comment by cjdeg** [comment]
@larouxn @tauthomas01, when can we expect this PR to be merged? Can we expect the payment method icon for our app to automatically switch to using the Credit Key icon?

**Comment by larouxn** [comment]
Hey all, Thomas and I no longer maintain this gem but glad to merge. I would expect this to be live within the month or so. 🤞

**Comment by cjdeg** [comment]
@larouxn @tauthomas01 @methodmissing Who is maintaining this gem? A month to deploy this is a really long time considering how long it's taken to get this merged. Our payment method is currently using the wrong payment icon.

**Comment by larouxn** [comment]
I will reach out to the team maintaining this going forward and let them know of the potential for a release.

**Pull request merged**